src/sage/calculus/transforms/dft.py

```diff
@@ -390,9 +401,9 @@ def dct(self):
         EXAMPLES::
 
             sage: J = list(range(5))
-            sage: A = [exp(-2*pi*i*I/5) for i in J]
-            sage: s = IndexedSequence(A,J)
-            sage: s.dct()
+            sage: A = [exp(-2*pi*i*I/5) for i in J]                                     # optional - sage.symbolic
```

Comment:
This one really uses the symbolic `pi` and `e`, but could easily be rewritten to use some other familiar sequence like the triangular numbers